### ISSUE TITLE
Earthquake Prediction Model Using Python _Phase 1 and Phase 2

### ISSUE THREAD

**Issue by VijayChandru06** [VijayPreeth/Earthquake-#1]
Earthquake Prediction Using Python
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.ensemble import RandomForestClassifier
from sklearn.metrics import accuracy_score

data = pd.read_csv('earthquake_data.csv')

x = data.drop('earthquake_occured',axis=1)
y = data['earthquake_occured']

x_train,x_test,y_train,y_test = train_test_split(x,y,test_size=0.2,random_state=42)
clf = RandomForestClassifier()

clf.fit(x_train,y_train)

y_pred = clf.predict(x_test)

accuracy = accuracy_score(y_test,y_pred)
print(f"Accuracy:{accuracy}")

**Comment by VijayChandru06** [comment]
![Screenshot_2023-10-10-23-07-48-55_c37d74246d9c81aa0bb824b57eaf7062](https://github.com/VijayPreeth/Earthquake-/assets/146064841/be022128-a7fd-4fc7-aab2-52951a613de9)
![Screenshot_2023-10-10-23-08-15-87_c37d74246d9c81aa0bb824b57eaf7062](https://github.com/VijayPreeth/Earthquake-/assets/146064841/30be3f4f-733e-447a-b42c-8ce4e3217f67)
